Lib/test/test_tokenize.py

```diff
@@ -1821,14 +1821,17 @@ def test_iter_compat(self):
 
 class TestRoundtrip(TestCase):
 
-    def check_roundtrip(self, f):
+    def check_roundtrip(self, f, *, compare_tokens_only=False):
         """
         Test roundtrip for `untokenize`. `f` is an open file or a string.
         The source code in f is tokenized to both 5- and 2-tuples.
         Both sequences are converted back to source code via
         tokenize.untokenize(), and the latter tokenized again to 2-tuples.
         The test fails if the 3 pair tokenizations do not match.
 
+        If `compare_tokens_only` is False, the exact output of `untokenize`
+        is compared against the original source code.
+
         When untokenize bugs are fixed, untokenize with 5-tuples should
         reproduce code that does not contain a backslash continuation
         following spaces.  A proper test should test this.
```

Comment:
I am not entirely sure what this is referring to. In any case, backslash continuation should now work in most cases, so maybe it can be removed?

Reply:
I think we can remove it if we know we have enough coverage with the changes in `check_roundtrip` (as I don't know if there are sufficient continuation lines around).